docs/interfaces/_types_.handler.html

```diff
@@ -3,8 +3,8 @@
 <head>
 	<meta charset="utf-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
-	<title>Handler | </title>
-	<meta name="description" content="Documentation for ">
+	<title>Handler | opine</title>
```

Comment:
Hadn't even noticed was missing this! Guessing is the `--name` passed to typedoc?

Reply:
Generating the docs is not as easy as it should be:

* `make typedoc` does not work, do you have it installed globally on your machine?

* `npx typedoc` does not work, can't find typescript


I've generated the docs with way:

1. npm init

2. npm i typedoc typescritp --save

3. run `npx --no-install <script in the makeFile, typedoc etc..>`

4. revert the deletion of the *.yaml

5. rm -rf node_modules/; rm ./package.json ./package-lock.json

6. run deno fmt otherwise make ci will fail due to a js file inside docs

Imo, this process could be improved and automated.

Reply:
For sure - it's non trivial, and yes I have it installed globally (to avoid having node_modules in the repo etc.)

I've raised an issue to get this automated --> #29 

Probs be the next thing I do as it's a real faff! Just been the only one working on this update to now so not _needed_ to improve it, but think it's worth doing 😄